### PR TITLE
W-15135730 GA branch for late Feb release jh

### PR DESCRIPTION
**Pull request by JennyHajee**

### CODE REVIEW

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 1]
```diff
@@ -0,0 +1,412 @@
+= Assigning Permissions 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: assigning-administrator-permissions.adoc
+
+Anypoint Platform comes with predefined permissions that you can set in Anypoint Access Management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
+
+The Salesforce platform uses the System Administrator profile which contains permissions to perform specific tasks in Salesforce. For API Experience Hub, this profile is required for the user who performs the Salesforce linking.
```
Comment (lucasaddisi) [inline modules/ROOT/pages/assigning-permissions.adoc]: >  For API Experience Hub, this profile is required for the user who performs the Salesforce linking.

From now on, the System Admin profile is not exclusively required to perform the linking. The Salesforce user could have either the System Admin Profile or the AEH_Anypoint Permission Set.

**Review by spezzirriemiliano — DISMISSED** [review]

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 1 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: Thanks for catching this @lucasaddisi. I update this sentence to add the permission set.

**Review by lucasaddisi — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/connecting-to-salesforce.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
...
+API Experience Hub has a Salesforce organization with Experience Cloud and a managed package. The managed package is installed during the onboarding process when the API Experience Hub administrator or a Salesforce user with the System Administrator profile or xref:user-roles-permissions-and-profiles.adoc[AEH_Anypoint permission set] connects their Salesforce organization to the Anypoint Platform organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization. The Salesforce System Administrator can view the *API Experience Hub* package in their Salesforce organization from Salesforce by navigating to *Setup* > *Installed Packages* and selecting the *API Experience Hub Package*.
```
Comment (hannanelson) [inline modules/ROOT/pages/connecting-to-salesforce.adoc]: has -> includes?

Comment (hannanelson) [reply modules/ROOT/pages/connecting-to-salesforce.adoc]: It's probably correct, but it feels weird to switch between their and your.

Comment (hannanelson) [reply modules/ROOT/pages/connecting-to-salesforce.adoc]: I'm surprised Vale didn't flag the first sentence as "long" 🤣

File: modules/ROOT/pages/connecting-to-salesforce.adoc [hunk 3]
```diff
@@ -34,13 +34,14 @@ After your styling changes for the portal are saved, the AEH Portal Admin team i
 
 == Before You Begin Connecting to Salesforce
 
-* Read the instructions in xref:prerequisites.adoc[Prerequisites].
-* Ensure that you are a System Admin in the Salesforce organization that you are connecting with.
+* Read the instructions in xref:before-you-begin-with-api-experience-hub.adoc[].
+* Ensure that you are a System Admin or have the AEH_Anypoint permission set in the Salesforce organization that you are connecting with.
```
Comment (hannanelson) [inline modules/ROOT/pages/connecting-to-salesforce.adoc]: Is System Admin the same as System Administrator on line 6?

File: modules/ROOT/pages/connecting-to-salesforce.adoc [hunk 4]
```diff
@@ -34,13 +34,14 @@ After your styling changes for the portal are saved, the AEH Portal Admin team i
...
 
 == Connect to Salesforce 
-The process of creating your API portal includes connecting with Salesforce Experience Cloud to link your Anypoint Platform organization to your Salesforce organization, then installing the Salesforce Experience Cloud package. The linking process is a one-time step when creating your portal.
+//The process of creating your API portal includes connecting with Salesforce Experience Cloud to link your Anypoint Platform organization to your Salesforce organization, then installing the Salesforce Experience Cloud package. 
+The linking process is a one-time step when creating your portal.
 
-NOTE: Only an Organization Administrator or API Experience Hub Administrator can link organizations.
+NOTE: Only an Organization Administrator, API Experience Hub Administrator, and a Salesforce user with the System Administrator profile or AEH_Anypoint permission set can link organizations.
```
Comment (hannanelson) [inline modules/ROOT/pages/connecting-to-salesforce.adoc]: should "and" be "or" here? As is, it could imply you need all three.

File: modules/ROOT/pages/getting-started-with-api-experience-hub.adoc [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
 endif::[]
+:page-aliases: prerequisites.adoc
 
 Verify that you have the following required accounts and access:
 
 * Anypoint Platform organization in production
 * Salesforce organization
 * Organization Administrator of the main organization
-* You must be a System Administrator in the Salesforce organization that you are linking to during the connection process
+* You must have the System Administrator profile or a Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process. For more information, see xref:assigning-permissions.adoc[].
```
Comment (hannanelson) [inline modules/ROOT/pages/getting-started-with-api-experience-hub.adoc]:
```suggestion
* You must have the System Administrator profile or be a Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process. For more information, see xref:assigning-permissions.adoc[].
```

Comment (hannanelson) [reply modules/ROOT/pages/getting-started-with-api-experience-hub.adoc]: Not sure if that's correct.

Comment (hannanelson) [reply modules/ROOT/pages/getting-started-with-api-experience-hub.adoc]: Does a profile get assigned? If so, then maybe something like:
* You must be assigned either the System Administrator profile ...

Comment (hannanelson) [reply modules/ROOT/pages/getting-started-with-api-experience-hub.adoc]: This is hard to parse. I wonder if a sublist is in order.

You must either be:
* Assigned the System Administrator profile
* A Salesforce user with the AEH_Anypoint permission set assigned in the Salesforce organization that you are connecting to during the linking process.

File: modules/ROOT/pages/getting-started-with-api-experience-hub.adoc [hunk 6]
```diff
@@ -22,10 +23,11 @@ To set up an organization, you can have the following Salesforce editions:
 * Enterprise Edition
 * Performance Editions of Salesforce
 
-If you do not have the required accounts and access, issues may occur during the Salesforce linking process when creating your API portal.
+If you do not have the required accounts and access, issues can occur during the Salesforce linking process when creating your API portal.
```
Comment (hannanelson) [inline modules/ROOT/pages/getting-started-with-api-experience-hub.adoc]:
```suggestion
If you don't have the required accounts and access, issues might occur during the Salesforce linking process when creating your API portal.
```

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 7]
```diff
@@ -0,0 +1,430 @@
+= Assigning Permissions 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: assigning-administrator-permissions.adoc
+
+Anypoint Platform comes with predefined permissions that you can set in Anypoint Access Management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
Anypoint Platform comes with predefined permissions that you can set in Anypoint Platform access management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
```

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#:~:text=control%20list%20(ACL)-,access%20management,-Not%20a%20title

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 8]
```diff
@@ -0,0 +1,430 @@
+= Assigning Permissions 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: assigning-administrator-permissions.adoc
+
+Anypoint Platform comes with predefined permissions that you can set in Anypoint Access Management to provide users with different access levels. You configure the permissions for administrators and Exchange viewers. 
+
+The Salesforce platform uses the System Administrator profile which contains permissions to perform specific tasks in Salesforce. For API Experience Hub, this profile or the AEH_Anypoint permission set is required for the user who performs the Salesforce linking.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
The Salesforce platform uses the System Administrator profile, which includes permissions to perform specific tasks in Salesforce. For API Experience Hub, this profile or the AEH_Anypoint permission set is required for the user who performs the Salesforce linking.
```

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: I feel like "AEH_Anypoint" should be in a different font - probably because the underbar makes it kind of codey.

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 9]
```diff
@@ -0,0 +1,430 @@
...
+
+
+== Before You Begin
+Consider the permission strategy for users in the Anypoint organization and Salesforce organization.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: Is there a link to the permission strategy? Or more information on this concept?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 10]
```diff
@@ -0,0 +1,430 @@
...
+
+== Grant the API Experience Hub Administrator Permission
+The Organization Administration and API Experience Hub Administrator can perform all of the functions to create a portal. Only the Organization Administrator or an existing API Experience Hub Administrator can grant a user the API Experience Hub Administrator permission. Grant this permission when you want another user in your organization to perform the Salesforce linking process, create and curate the portal, manage the portal, and manage users.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc:18]: Is "Organization Administration" correct or should it be [Administrator](https://docs.mulesoft.com/access-management/permissions-by-product#:~:text=Access%20Management-,Organization%20Administrator,-At%20the%20root)?

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc:18]: [API Experience Hub Admin](https://docs.mulesoft.com/access-management/permissions-by-product#api-experience-hub:~:text=API%20Experience%20Hub%20Admin)?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 11]
```diff
@@ -0,0 +1,430 @@
...
+
+To grant the API Experience Hub Administrator permission to a user:
+
+. Log in to Anypoint Platform using an account with the Organization Administrator permission.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
. Sign in to Anypoint Platform using an account with the Organization Administrator permission.
```

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: The [Anypoint Platform](https://anypoint.mulesoft.com/login/signin) page says Sign in.

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 10 above)
Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc:18]: If ☝️ correct, apply globally.

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 12]
```diff
@@ -0,0 +1,430 @@
...
+. Click *Access Management*.
+. Click *Users*.
+. Click the username of the user that you want to add the API Experience Hub Administrator permission. 
+. Select *Permissions* and click *Add Permissions*.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
. Select *Permissions* and click *Add permissions*.
```

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: (lc in UI)

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 13]
```diff
@@ -0,0 +1,430 @@
...
+. From the *Select permissions to add for this user* search box, enter *API Experience Hub Amin* and select the checkbox.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
. In the *Select permissions to add for this user* search box, enter *API Experience Hub Admin* and select the checkbox.
```

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: I get no results, probably because I don't have the right permissions. Would it be worth adding a note?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 14]
```diff
@@ -0,0 +1,430 @@
...
+. Click the blue plus icon.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: In my UI, I don't have this - I have the Next button, though. I think I opted for the new UI, but maybe not?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 15]
```diff
@@ -0,0 +1,430 @@
...
+//. Click the username of the user that you want to add the API Experience Hub Administrator permission. 
+//. Select *Permissions* and click *Add Permissions*.
+//. From the *Select permissions to add for this user* search box, enter *API Experience Hub Portal Amin* and select the checkbox.
+//. Click the blue plus icon.
+
+== Grant the Exchange Viewer Permission
+
+Administrators can only add APIs for which they have the Exchange Viewer permission. All administrators must have the Exchange Viewer permission to add APIs to the portal.
+
+When APIs are added to the portal, all administrators are assigned automatically the Exchange Viewer permission for the APIs added in the portal.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: Any way to streamline this? 

When APIs are added to the portal.... the APIs added in the portal.

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 16]
```diff
@@ -0,0 +1,430 @@
...
+
+//. From *Access Management*, click *Users*.
+//. Click the username of the user that you want to add the API Experience Hub Administrator permission. 
+//. Select *Permissions* and click *Add Permissions*.
+//. From the *Select permissions to add for this user* search box, enter *API Experience Hub Portal Amin* and select the checkbox.
+//. Click the blue plus icon.
+
+== Grant the Exchange Viewer Permission
+
+Administrators can only add APIs for which they have the Exchange Viewer permission. All administrators must have the Exchange Viewer permission to add APIs to the portal.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: Are these two sentences saying the same thing?

To add APIs to the portal, administrators must have Exchange Viewer permissions for those APIs.

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 17]
```diff
@@ -0,0 +1,430 @@
...
+
+When APIs are added to the portal, all administrators are assigned automatically the Exchange Viewer permission for the APIs added in the portal. 
+
+To grant the Exchange Viewer permission to a user:
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: a user or administrator?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 18]
```diff
@@ -0,0 +1,430 @@
...
+
+. From *Access Management*, click *Users*.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: This seems largely the same as the previous procedure. Any way to combine? 
Or actually, do you need to document this here or can you point to the access management doc for the [steps](https://docs.mulesoft.com/access-management/users#grant-user-permissions:~:text=To%20grant%20permissions%20to%20an%20individual%20user%20in%20the%20legacy%20roles%20interface%3A)? 

Typically, we just list the permissions required and point to AM for the details. See https://docs.mulesoft.com/mq/mq-access-management#give-users-anypoint-mq-access-permissions and
https://docs.mulesoft.com/cloudhub-2/ps-assign-permissions

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 19]
```diff
@@ -0,0 +1,430 @@
...
+. Click the username of the user that you want to add the Exchange Viewers permission. 
+. Select *Permissions* and click *Add Permissions*.
+. From the *Select permissions to add for this user* search box, enter *Exchange Viewers* and select the checkbox.
+. Click the blue plus icon.
+
+== Create the AEH_Anypoint Permission Set With the Minimum Permissions for the Salesforce User To Perform the Linking Process
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
== Create the AEH_Anypoint Permission Set with the Minimum Permissions for the Salesforce User to Perform the Linking Process 
```

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: or 

Create the AEH_Anypoint Permission Set with the Minimum Permissions to Link Organizations

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 20]
```diff
@@ -0,0 +1,430 @@
...
+. Click the blue plus icon.
+
+== Create the AEH_Anypoint Permission Set With the Minimum Permissions for the Salesforce User To Perform the Linking Process 
+
+//Apply the Principle of Least Privilege for Your Salesforce Organization
+A Salesforce user with the System Administrator profile is required to connect the Salesforce Experience Cloud and link the Salesforce organization with the Anypoint Platform organization. The System Administrator profile may have more permissions than necessary. For users with an existing Salesforce organization and potentially sensitive data, ensure the security of that data by following the principle of least privilege. Use the AEH_Anypoint permission set to configure only those permissions for the user who performs the linking process. The Salesforce user with the System Administrator profile creates the AEH_Anypoint permission set with all the necessary permissions and assigns the permission set to a Salesforce user. You can create the permission set manually in Salesforce or deploy it via your CI/CD pipeline using the *AEH_Anypoint.permissionset-meta.xml* metadata file.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: Is Salesforce Experience Cloud a product name? If so, then no article ("the").

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: may -> might

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: The switch between third- and second-person feels weird to me. 

Is "The Salesforce user" the same as "You"?

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: Don't use bold for [filenames](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=File%20names%2C%20including,Mono).

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 21]
```diff
@@ -0,0 +1,430 @@
...
+
+The permissions for the System Admin can be downgraded with the permissions set before or after there is a connection to Salesforce. If the Salesforce user is the same user who created the connection, then that user can be assigned to the AEH_Anypoint permission set without disconnecting the portal and creating the connection again. If you want to assign the permission set to another Salesforce user who did not create the connection, the portal must be disconnected and created again. For example, the original Salesforce user who connected to Salesforce left the company.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: System Admin = System Administrator?

Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
You can downgrade the permissions for the System Admin using the permissions set before or after connecting to Salesforce. If the Salesforce user is the same user who created the connection, you can assign that user to the AEH_Anypoint permission set without disconnecting the portal and recreating the connection. To assign the permission set to Salesforce user who didn't create the connection, you must disconnect the portal and recreate the connection. For example, the original Salesforce user who connected to Salesforce left the company.
```

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 22]
```diff
@@ -0,0 +1,430 @@
...
+
+Perform the following steps to manually create the permission set, add the necessary permissions, and assign a user to the permission set:
+
+. Create the AEH_Anypoint permission set:
++
+.. Log in to Salesforce as the System Administrator.
+.. In *Setup*, enter `permission sets` in the Quick Find box and click *Permission Sets*.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
.. In *Setup*, enter `permission sets` in the *Quick Find* box and click *Permission Sets*.
```

Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: (if Quick Find is in the UI)

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 23]
```diff
@@ -0,0 +1,430 @@
...
+.. Click *Add*.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc:62]: Any way to collapse some of these steps?

Enter `AEH_Anypoint` in both the *Label* and *API Name* fields and click *Save*.
...click *System Perminssions*, and then click *Edit*.

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 24]
```diff
@@ -0,0 +1,430 @@
...
++
+* Apex REST Services
+* API Enabled
+* Assign Topics
+* Author Apex
+* Connect Organization to Environment Hub
+* Create and Customize Dashboards
+* Create and Customize List Views
+* Create and Customize Reports
+*  Create and Set Up Experiences
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: Does this render correctly with the extra space?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 25]
```diff
@@ -0,0 +1,430 @@
...
+* Customize Application
+* Delete Topics
+* Download AppExchange Packages
+* Edit Events
+* Edit HTML Templates
+* Edit My Dashboards
+* Edit My Reports
+* Edit Tasks
+* Edit Topics
+* Manage Auth. Providers
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc:91]: Does the UI have the period?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 26]
```diff
@@ -0,0 +1,430 @@
...
+* Manage Roles
+* Manage Sharing
+* Manage Users
+* Reset User Passwords and Unlock Users
+* View All Users
+.. Enable any other permissions that you want to add to the permission set.
+.. Click *Save*.
+. Assign the permission set to a Salesforce user:
++
+.. In *Setup*, enter `Users` in the Quick Find box and click *Permission Sets*, and click *Users*.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: Quick Find in UI?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 27]
```diff
@@ -0,0 +1,430 @@
...
+.. In the *Full Name* column, select the name of the user.
+.. In *Permission Set Assignments* section, click *Edit Assignments*.
+.. Select *AEH_Anypoint* from the *Available Permission Sets* list and move the set to the *Enabled Permission Sets* list.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc:134]: Can you just say "Move *AEH_Anypoint* from the *Available Permission Sets* list to the *Enabled Permission Sets* list."?

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 28]
```diff
@@ -0,0 +1,430 @@
...
+.. Click *Save*.
+
+To deploy the permission set via your CI/CD, use the following AEH_Anypoint.permissionset-meta.xml metadata file:
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
To deploy the permission set via your CI/CD, use the following `AEH_Anypoint.permissionset-meta.xml` metadata file:
```

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 29]
```diff
@@ -0,0 +1,430 @@
...
+
+. Copy the metadata file and create a file named *AEH_Anypoint.permissionset-meta.xml*.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]:
```suggestion
. Copy the metadata file and create a file named `AEH_Anypoint.permissionset-meta.xml`.
```

File: modules/ROOT/pages/assigning-permissions.adoc [hunk 30]
```diff
@@ -0,0 +1,430 @@
...
+. Run the following command and replace `path/to/permissionset/` and `orgAlias` variables with your path and org alias:
++
+[source,code]
+----
+sf project deploy start --source-dir path/to/permissionset/ -o orgAlias
+----
+. To verify that the permission set was created:
++
+.. Log in to Salesforce as the System Administrator.
+.. In *Setup*, enter `permission sets` in the Quick Find box and click *Permission Sets*.
```
Comment (hannanelson) [inline modules/ROOT/pages/assigning-permissions.adoc]: *Quick Find*

File: modules/ROOT/pages/connecting-to-salesforce.adoc (hunk 3 above)
Comment (JennyHajee) [reply modules/ROOT/pages/connecting-to-salesforce.adoc]: yes

File: modules/ROOT/pages/connecting-to-salesforce.adoc (hunk 4 above)
Comment (JennyHajee) [reply modules/ROOT/pages/connecting-to-salesforce.adoc]: yes

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 8 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: Which font do you suggest?

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 10 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc:18]: Organization Administrator is correct and so is API Experience Hub Administrator.

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 11 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: thanks

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 13 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: The intro section says that only the Org admin or existing AEH admin can grant this. I will add this to Before You Begin as well.

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 14 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: Thanks for pointing out the new UI. I updated the steps.

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 17 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: an admin

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 18 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: I'll combine those for now.

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 21 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: System Administrator

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 24 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc]: extra space, will remove it

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 25 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc:91]: yes

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 27 above)
Comment (JennyHajee) [reply modules/ROOT/pages/assigning-permissions.adoc:134]: This is how it's documented in SF. I think it is ok.

File: modules/ROOT/pages/connecting-to-salesforce.adoc [hunk 31]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
...
+API Experience Hub contains a Salesforce organization with Experience Cloud and a managed package. The managed package is installed during the onboarding process. The API Experience Hub administrator or a Salesforce user with the System Administrator profile or xref:user-roles-permissions-and-profiles.adoc[AEH_Anypoint permission set] connects the Salesforce organization to the Anypoint Platform organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization. The Salesforce System Administrator can view the *API Experience Hub* package in the Salesforce organization from Salesforce by navigating to *Setup* > *Installed Packages* and selecting the *API Experience Hub Package*.
```
Comment (hannanelson) [inline modules/ROOT/pages/connecting-to-salesforce.adoc:6]:
```suggestion
API Experience Hub contains a Salesforce organization with Experience Cloud and a managed package. The managed package is installed during the onboarding process. The API Experience Hub administrator or a Salesforce user with the System Administrator profile or xref:user-roles-permissions-and-profiles.adoc[AEH_Anypoint permission set] connects the Salesforce organization to the Anypoint Platform organization. The managed package contains the components and data objects that interact with your Anypoint Platform organization. The Salesforce System Administrator can view the *API Experience Hub* package in the Salesforce organization from Salesforce by navigating to *Setup* > *Installed Packages* and selecting the *API Experience Hub Package*.
```

File: modules/ROOT/pages/assigning-permissions.adoc (hunk 8 above)
Comment (hannanelson) [reply modules/ROOT/pages/assigning-permissions.adoc]: Usually, we either put it in mono font or remove the underbar, but if that's how Salesforce handles permission sets, then it's fine.

**Review by hannanelson — APPROVED** [review]
